### PR TITLE
SDK update for Azure Backup api version 2019-05-13

### PR DESCRIPTION
**Pull request by SanjayHukumRana**
Swagger PR for api version 2019-05-13 : https://github.com/Azure/azure-rest-api-specs/commit/e133429d289630c215c49580ab06714905248238

### CODE REVIEW

File: sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/Microsoft.Azure.Management.RecoveryServices.Backup.csproj [hunk 1]
```diff
@@ -6,13 +6,16 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.RecoveryServices.Backup</PackageId>
 		<Description>Provides developers with libraries for the updated recovery services backup feature under Azure Resource manager.</Description>
-		<Version>3.1.2-preview</Version>
+		<Version>3.1.3-preview</Version>
```
Comment (isra-fel) [inline sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/Microsoft.Azure.Management.RecoveryServices.Backup.csproj]: I would suggest use 3.2.0 or even 4.0.0 for such a big change, especially you switched to a new api version

**Review by isra-fel — CHANGES_REQUESTED** [review]
Please also fix CI errors

File: sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/Microsoft.Azure.Management.RecoveryServices.Backup.csproj (hunk 1 above)
Comment (SanjayHukumRana) [reply sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/Microsoft.Azure.Management.RecoveryServices.Backup.csproj]: Done

**Comment by SanjayHukumRana** [comment]
Hi @Yeming Liu<mailto:Yeming.Liu@microsoft.com>,

PR is fixed with comments and CI fixes.
Please review.

Thanks.

**Review by isra-fel — APPROVED** [review]

**Pull request merged**